docs/generators/MSSqlDDL.md

```diff
@@ -0,0 +1,494 @@
+|SEQUENCE                      |INTEGER IDENTITY(1,1)          |hasSequenceReturning == True |SEQUENCE      |INTEGER IDENTITY(1,1)|
+|BIGSEQUENCE                   |BIGINT                         |hasSequenceReturning == False|BIGSEQUENCE   |BIGINT               |
+|BIGSEQUENCE                   |BIGINT IDENTITY(1,1)           |hasSequenceReturning == True |BIGSEQUENCE   |BIGINT IDENTITY(1, 1)|
+|DATE                          |DATETIME                       |                             |DATE          |DATETIME             |
+|DATETIME                      |DATETIME                       |                             |DATETIME      |DATETIME             |
+|BLOB                          |IMAGE                          |                             |BLOB          |IMAGE                |
+|BIGXML                        |XML                            |                             |BIGXML        |XML                  |
+|XML                           |XML                            |                             |XML           |XML                  |
+|MONEY                         |MONEY                          |                             |MONEY         |MONEY                |
+|USERSTAMP                     |VARCHAR(50)                    |                             |USERSTAMP     |VARCHAR(50)          |
+|UID                           |UNIQUEIDENTIFIER               |                             |UID           |UNIQUEIDENTIFIER     |
+|undefined SI type             |unkown                         |                             |*             |unknown              |
+
+The `hasSequenceReturning` variable is `False` by defualt and is set to `True` if the standard INSERT proc has the RETURNING keyword:
+```
+PROC INSERT RETURNING
+```
+
+# Automatic Adding of Fields
+## TmStamp
+The TmStamp field can automatically be added to the table by specifying the `--flag "add timestamp"` flag:
+```
+TABLE Example
+    ID                  SEQUENCE
+    SOME_DATA           CHAR(100)
+    USER_ID             USERSTAMP
+```
+Jportal Params:
+```
+--builtin-generator MSSqlDDL:\tmp --flag "add timestamp"
+```
+Example output:
+```
+CREATE TABLE ExampleSchema.Example
+(
+  ID INTEGER IDENTITY(1,1) NOT NULL
+, SOME_DATA VARCHAR(100) NOT NULL
+, USER_ID VARCHAR(50) NOT NULL
+, TIMESTAMP
+, CONSTRAINT PK_ExampleSchema_Example_PKEY PRIMARY KEY (
+    ID
+  )
+)
+GO
+```
+
+## UserId and TmStamp
+UserId's and TmStamp fields can automatically be added to the table by specifing `--flag "internal stamps"` to the generator.
+```
+TABLE Example
+    ID                  SEQUENCE
+    SOME_DATA           CHAR(100)
+```
+Jportal Params:
+```
+--builtin-generator MSSqlDDL:\tmp --flag "internal stamps"
+```
+Example output:
+```
+CREATE TABLE ExampleSchema.Example
+(
+  ID INTEGER IDENTITY(1,1) NOT NULL
+, SOME_DATA VARCHAR(100) NOT NULL
+, UpdateWhen  DATETIME DEFAULT CURRENT_TIMESTAMP NULL
```

Comment:
How does one specify the columns names? UpdatedWhen and UpdatedByWho doesn't conform to any of the existing project standards.
Standards currently are
USERID
TMSTAMP

or 
last_updated
last_updated_user

Reply:
Cool.

Reply:
#66